### PR TITLE
Decouple kronjob

### PR DESCRIPTION
**Pull request by liambuchanan**
This will allow the kronjob api to move independently

### CODE REVIEW

File: cronner/kronjob_util.py [hunk 1]
```diff
@@ -19,12 +18,11 @@ def validate_kronjob_entry(entry):
         raise KronjobValidationException('entry names must not be longer than 52 chars. ({})'.format(entry['name']))
 
 
-def serialize_kronjob(base_template, entries):
+def serialize_kronjob(base_template, entries)
```
Comment (archerabi) [inline cronner/kronjob_util.py]: hmm, is this valid syntax without the : ?

**Review by archerabi — APPROVED** [review]
with comment

File: cronner/kronjob_util.py (hunk 1 above)
Comment (liambuchanan) [reply cronner/kronjob_util.py]: 😬

**Comment by liambuchanan** [comment]
thanks @archerabi

**Pull request merged**